test/unit/math/mix/mat/fun/resize_test.cpp

```diff
@@ -8,12 +8,12 @@ using stan::math::var;
 
 TEST(AgradMixMatrixResize, fvar_var) {
   fvar<var> x = 5;
-  std::vector<size_t> dims;
+  std::vector<int> dims;
```

Comment:
Again, I'm just changing the resize tests to work with ints and use 2 dimensional sizes for Vector and RowVector.